docker/data/zkevm/node.toml

```
@@ -32,8 +32,8 @@ Outputs = ["stderr"]
 FreeClaimGasLimit = 1500000
 IntervalToRefreshBlockedAddresses = "5m"
 IntervalToRefreshGasPrices = "5s"
-MaxTxBytesSize = 100132
-MaxTxDataBytesSize = 100000
+MaxTxBytesSize=100132
+MaxTxDataBytesSize=100000
```

Nitpick, to have consistent formatting
```suggestion
MaxTxBytesSize = 100132
MaxTxDataBytesSize = 100000
```

---

Comments: https://github.com/0xPolygon/agglayer/pull/90/commits/986390a1c514a7a2bdc6730c34a9c9e78f3d9bd1